charts/kube-prometheus-stack/values.yaml

```diff
@@ -1435,6 +1435,21 @@ prometheusOperator:
     sha: ""
     pullPolicy: IfNotPresent
 
+  ## Prometheus image to use for prometheuses managed by the operator
+  ##
+  # prometheusDefaultBaseImage: quay.io/prometheus/prometheus
+
+  ## Alertmanager image to use for alertmanagers managed by the operator
+  ##
+  # alertmanagerDefaultBaseImage: quay.io/prometheus/alertmanager
+
+  ## Configmap-reload image to use for reloading configmaps
```

Comment:
This should probably not be readded here.

Reply:
Are you asking me to remove it from the values.yaml file or place it elsewhere in the file?

Reply:
The configmap-reload image got removed with the upgrade to prometheus-operator v0.45.0, so there's no need to add it back.

Reply:
Thanks, I missed that during the rebase 🤦🏻